roles/k3s/master/tasks/main.yml

```diff
@@ -19,6 +19,13 @@
 - name: Deploy metallb manifest
   include_tasks: metallb.yml
 
+- name: Create node password file if not exist
+  lineinfile:
+    path: /etc/rancher/node/password
+    line: "{{ (node_password_salt | default('') + ansible_hostname) | md5 }}"
```

Comment:
That seems really risky to me! If someone forgets to define a value for the salt, we will have none. Should we default this to a random string instead?